evalml/data_checks/highly_null_data_check.py

```diff
@@ -171,3 +160,26 @@ def validate(self, X, y=None):
                 ).to_dict()
             )
         return results
+
+    @staticmethod
+    def get_null_column_information(X, pct_null_col_threshold=0.0):
+        """Finds columns that are considered highly null (percentage null is greater than threshold) and returns dictionary mapping column name to percentage null and dictionary mapping column name to null indices."""
```

Comment:
Worth it to add `Arguments` and `Returns`? (for this and `get_null_row_information`)